packages/jazz-tools/src/coValues/account.ts

```diff
@@ -288,11 +288,15 @@ export class Account extends CoValueBase implements CoValue {
 
     as._raw.core.node.syncManager.addPeer(connectedPeers[1]);
 
-    return this.create<A>({
+    const account = await this.create<A>({
       creationProps: options.creationProps,
       crypto: as._raw.core.node.crypto,
       peersToLoadFrom: [connectedPeers[0]],
     });
+
+    await account.waitForAllCoValuesSync();
+
+    return account;
```

Comment:
Needed this change because the content reconciliation now requires more than one roundtrip.

Reply:
thanks for catching it